ConfigSpace/hyperparameters.pyx

```diff
@@ -700,16 +751,25 @@ cdef class NormalFloatHyperparameter(FloatHyperparameter):
             sigma=self.sigma,
             log=self.log,
             q=self.q,
+            lower=self.lower,
+            upper=self.upper,
             meta=self.meta
         )
 
     def __hash__(self):
         return hash((self.name, self.mu, self.sigma, self.log, self.q))
 
     def to_uniform(self, z: int = 3) -> 'UniformFloatHyperparameter':
+        if self.lower is not None and self.upper is not None:
+            lb = self.mu - (z * self.sigma)
+            ub = self.mu + (z * self.sigma)
+        else:
+            lb = self.lower
+            ub = self.upper
+
```

Comment:
One or both of `self.lower` and `self.upper` are None here. I imagine the if statement should read:
```